_episodes/05-dockerfiles.md

```diff
@@ -118,40 +123,58 @@ scikit-learn        1.3.1
 ~~~
 {: .output}
 
+You can list all images available on your local machine with `podman images`:
+~~~bash
+podman images
+~~~
+{: .source}
+
+~~~
+REPOSITORY                                 TAG            IMAGE ID      CREATED       SIZE
+localhost/extend-example                   latest         c24a757fabe7  8 hours ago   2.2 GB
+docker.io/matthewfeickert/intro-to-docker  latest         64708e04f3a9  2 years ago   1.62 GB
+...
+~~~
+{: .output}
+
+`docker.io` indicates that the image was pulled from the Docker Hub,
+while `localhost` indicates that the image was built locally.
+
 ## Tags
 
 In the examples so far the built image has been tagged with a single tag (e.g. `latest`).
 However, tags are simply arbitrary labels meant to help identify images and images can
 have multiple tags.
-New tags can be specified in the `docker build` command by giving the `-t` flag multiple
+New tags can be specified in the `podman build` (or `docker build`) command by giving the `-t` flag multiple
 times or they can be specified after an image is built by using
-[`docker tag`][docker-docs-tag].
+[`podman tag`][podman-docs-tag].
 
 ~~~bash
-docker tag <SOURCE_IMAGE[:TAG]> <TARGET_IMAGE[:TAG]>
+podman tag <SOURCE_IMAGE[:TAG]> <TARGET_IMAGE[:TAG]>
 ~~~
 {: .source}
 
 > ## Add your own tag
 >
-> Using `docker tag` add a new tag to the image you built.
+> Using `podman tag` add a new tag to the image you built.
 >
 > > ## Solution
 > >
 > > ~~~
-> >docker images extend-example
-> >docker tag extend-example:latest extend-example:my-tag
-> >docker images extend-example
+> >podman images extend-example
+> >podman tag extend-example:latest extend-example:my-tag
+> >podman images extend-example
 > > ~~~
 > > {: .source}
 > >
 > > ~~~
-> >REPOSITORY          TAG                 IMAGE ID            CREATED            SIZE
-> >extend-example      latest              b571a34f63b9        t seconds ago      1.59GB
+> >REPOSITORY                TAG         IMAGE ID      CREATED      SIZE
+> >localhost/extend-example  latest      c24a757fabe7  9 hours ago  2.2 GB
 > >
-> >REPOSITORY          TAG                 IMAGE ID            CREATED            SIZE
-> >extend-example      latest              b571a34f63b9        t seconds ago      1.59GB
-> >extend-example      my-tag              b571a34f63b9        t seconds ago      1.59GB
+> >REPOSITORY                TAG         IMAGE ID      CREATED      SIZE
+> >localhost/extend-example  my-tag      c24a757fabe7  9 hours ago  2.2 GB
+> >localhost/extend-example  latest      c24a757fabe7  9 hours ago  2.2 GB
+
 > > ~~~
 > > {: .output}
 > {: .solution}
```

Comment:
This section does not format correctly on the web page.

Reply:
Oh yes, a missing `> >`. Fixed in the coming commit